cttportfolio.css

```diff
@@ -1,470 +1,874 @@
-	padding-bottom:2%;
-}
-.project_main_links .project_icon_size{
-	width:8%;
-}
-.pf_top button{
-	margin-top:80px;
-	margin-bottom:80px;
+	align-items: flex-end;
+	margin-top: 100px;
+}
+
+.project_container>* {
+	height: 100%;
+}
+
+.project_img {
+	display: block;
+	height: 100%;
+	max-width: 100%;
+	object-fit: cover;
 }
+
+.pf_link {
+	width: 30%;
+	object-fit: cover;
+}
+
+.project_text {
+	text-decoration: none;
+	color: rgb(2, 7, 9);
+	font-size: 2rem;
+	width: 70%;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	border-radius: 5px;
+}
+
+.project_text>* {
+	padding: 20px;
+}
+
+.project_text h5 {
+	margin: 0;
+	align-self: flex-start;
+}
+
+.project1 .project_text {
+	margin-left: 50px;
+}
+
+.project2 .project_text {
+	margin-right: 50px;
+}
+
+.project2 h5 {
+	align-self: flex-end;
+}
+
+.project2 .project_links {
+	align-self: flex-start;
+}
+
+.project_main_links {
+	width: 70%;
+	align-self: flex-end;
+	display: flex;
+	justify-content: space-between;
+	padding-bottom: 2%;
+}
+
+.project_main_links .project_icon_size {
+	width: 8%;
+}
+
+.pf_top button {
+	margin-top: 80px;
+	margin-bottom: 80px;
+}
+
+
 /*======================================== SUB PORTFOLIO SECTION ========================================*/
-.pf_sub{
-	display:flex;
-	justify-content:space-between;
-	height:500px;
-}
-.pf_sub > a {
-	width:30%;
-	height:100%;
-	text-decoration:none;	
-}
-.pf_sub_link_div{
-	height:100%;
-	display:flex;
-	flex-direction:column;
-	align-items:center;
-}
-.pf_sub_link_div > img{
-	object-fit:cover;
-	height:100%;
-	max-width:100%;
-}
-.pf_sub h6{
-	text-decoration:none;
-	font-weight:600;
-	font-size:2.0rem;
-	padding:5%;
+
+.pf_sub {
+	display: flex;
+	justify-content: space-between;
+	height: 500px;
+}
+
+.pf_sub>a {
+	width: 30%;
+	height: 100%;
+	text-decoration: none;
+}
+
+.pf_sub_link_div {
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.pf_sub_link_div>img {
+	object-fit: cover;
+	height: 100%;
+	max-width: 100%;
+}
+
+.pf_sub h6 {
+	text-decoration: none;
+	font-weight: 600;
+	font-size: 2.0rem;
+	padding: 5%;
 }
+
+
 /*======================================== CONTACT SECTION ========================================*/
-.contactme{
-	display:flex;
-	align-items:center;
-	flex-direction:column;
-	margin-top:100px;
-}
-.contactmetext{
-	display:flex;
-	flex-direction:column;
-	justify-content:space-between;
-	align-items:center;
-	width:100%;
-}
-.contactmetext > div{
-	margin:50px 0px;
-	padding:50px 50px;
-}
-.contact_div{
-	display:flex;
-	justify-content:space-between;
-	align-items:center;
-	width:80%;
-}
-.contact_div > *{
-	margin:0;
-}
-.contact_div h6{
-	font-size:2rem;
-}
-.contact_input{
-	width:50%;
+
+.contactme {
+	display: flex;
+	align-items: center;
+	flex-direction: column;
+	margin-top: 100px;
 }
+
+.contactmetext {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	align-items: center;
+	width: 100%;
```

Comment:
Elements which have a `display` type of `block` have a `width` of `100%` by default, therefore this does nothing.

Reply:
it doesnt show but the display here is actually flexbox

Reply:
Yeah, still applys as `flex` inherits features of `block` where as `flex-inline` inherits features of `inline`. :)